README.md

```diff
@@ -93,6 +94,23 @@ Finally, install the dependencies:
 npm install
 ```
 
+## Faster development building with ic-design-system
+
+If you want to build the components in this repository and use it in [`ic-design-system`](https://github.com/mi6/ic-design-system), you can use `npm link` in `ic-design-system`. This will speed up development because you do not have to generate a compressed npm package, but rather directly use the package after building it locally.
+
+First, build the monorepo packages in the top-level directory of `ic-ui-kit`:
+
+```console
+npm run build
+```
+
+Then to use the package, make sure you are in the top-level directory of `ic-design-system`. You will need to link the generated build from `ic-ui-kit`: 
+
+```console
+npm link ../path/to/ic-ui-kit/packages/react
+```
+> [!NOTE]
+> Now you can develop and debug faster with `ic-ui-kit` in `ic-design-system`.
```

Comment:
Do we need these two lines? I don't think they add anything that useful but just my opinion :)

Reply:
It's not essential although reading this from the perspective of a new contributor, this is sort of a message saying "All the steps are done, now you can start your work"